### PR TITLE
 Utility classes should not have public constructors

### PR DESCRIPTION
**Pull request by AymanDF**
This pull request is focused on resolving occurrences of Sonar rule squid:S1118 - “Utility classes should not have public constructors ”. You can find more information about the issue here: 
https://dev.eclipse.org/sonar/rules/show/squid:S1118
Please let me know if you have any questions.
Ayman Abdelghany.

### CODE REVIEW

**Comment by metc** [comment]
Hi @AymanDF 
Could you please rebase with the `1.7.10-MNA` branch ?
I fixed our CI system today. We would like to do some checks before merging. Thanks !

File: src/main/java/mods/eln/client/SoundLoader.java [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 public class SoundLoader {
 
+	private SoundLoader() {
```
Comment (metc) [inline src/main/java/mods/eln/client/SoundLoader.java]: Some default constructors are needed in order to be registered in the event bus

Comment (AymanDF) [inline src/main/java/mods/eln/client/SoundLoader.java]: Fixed :)

**Comment by metc** [comment]
Thanks @AymanDF !

**Pull request merged**